src/static/themes/dark/content.css

```diff
@@ -0,0 +1,12 @@
+:root.dark {
```

Comment:
`:root.dark` is fine for pages we control, but on web content we need to work a bit harder to avoid collisions with page classes.

To avoid writing out `:root.BigTridactylThemePrefixDark` each time (if the theme does more than change the variables) we can use sass or something similar.